seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -5202,6 +5202,7 @@
       ],
       "tests": [
         "assert(code.match(/for\\s*\\(/g).length > 1, 'message: You should be using a <code>for</code> loop for this.');",
+        "assert(code.match(/push\\s*\\(/g).length > 1, 'message: You should be using <code>push</code> like the example.');",
```

Comment:
If I write `push` in a comment and still use `unshift`, the test will pass. You can make the regExp more specific by trying to match `myArray.push`.